CoastalResilienceNetwork/GeositeFramework
# Merge static site branch into develop branch

## Overview

Merges the static site branch into the `develop` branch.

## Testing instructions

- Checkout this branch, and verify the .NET site is gone and the static site is present.
- Try to build and run the site, and verify it works.
